src/components/networked.js

```diff
@@ -162,6 +156,34 @@ AFRAME.registerComponent('networked', {
     if (this.isMine() && this.needsToSync()) {
       this.syncDirty();
     }
+
+    var now = Date.now();
+
+    if (!this.isMine()) {
+      for (const posComp of this.positionComponents) {
+        var progress = (now - posComp.lastUpdated) / posComp.duration;
+        
+        if (progress <= 1) {
```

Comment:
Looks like this actually wont ever fire on the initial update since progress ends up being > 1 (from the duration being 1ms)